Firestore/core/src/firebase/firestore/util/log_apple.mm

```diff
@@ -50,11 +50,11 @@ FIRLoggerLevel ToFIRLoggerLevel(LogLevel level) {
 void LogMessageV(LogLevel level, NSString* format, ...) {
   va_list list;
   va_start(list, format);
-
-  FIRLogBasic(ToFIRLoggerLevel(level), kFIRLoggerFirestore, @"I-FST000001",
-              format, list);
-
+  NSString* formattedMessage = [[NSString alloc] initWithFormat:format
```

Comment:
Please update this call - it's causing `pod lib lint` to fail with no matching API call.